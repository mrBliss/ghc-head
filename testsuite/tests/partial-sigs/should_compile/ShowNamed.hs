{-# LANGUAGE PartialTypeSignatures #-}
module ShowNamed where
       
showTwo :: Show _a => _a -> String
showTwo x = show x

