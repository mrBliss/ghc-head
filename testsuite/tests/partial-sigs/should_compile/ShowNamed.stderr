TYPE SIGNATURES
  showTwo :: forall tw_a. Show tw_a => tw_a -> String
TYPE CONSTRUCTORS
COERCION AXIOMS
Dependent modules: []
Dependent packages: [base, ghc-prim, integer-gmp]
